suite-common/wallet-config/src/networksConfig.ts

```diff
@@ -435,3 +435,10 @@ export const getTestnets = (debug = false) =>
     networksCompatibility.filter(
         n => !n.accountType && n.testnet === true && (n.symbol !== 'regtest' || debug),
     );
+
+// Checks if the network uses Ethereum-like address logic.
+// Eg. uses a receive address instead of a XPUB key for the import.
+export const isEthereumLikeAddressNetwork = (symbol: NetworkSymbol) => {
```

Comment:
I probably don't like this. "ETH like !== XRP like" so name itself is misleading. It shoudn't be placed in common because it is not used in desktop a probably won't be. 

From usage I would prefer map approach:

```
const buttonMessagesForNetworkType: Record<NetworkType, string> = {
   bitcoin: "Message for BTC",
   ethereum: "Message for ETH",
   ...
}
```
You will need to define all types but this approach is much more better if we for example add another network type, because it will throw there is missing case and you can fix it easily and also you wont need to do "if-else" spaggeti code for displayin message.

Reply:
changed in 72f5f54b5